backend/src/controllers/controller.ts

```diff
@@ -0,0 +1,29 @@
+import { Request, Response } from 'express';
```

Comment:
What is this controller?

Reply:
this controller was for testing purposes only, removed.